bin/templates/scripts/cordova/lib/projectFile.js

```diff
@@ -40,8 +40,19 @@ function parseProjectFile (locations) {
     const xcodeproj = xcode.project(pbxPath);
     xcodeproj.parseSync();
 
+    let projectName = '';
+    const workspaceCollection = fs.readdirSync(project_dir).find(d => d.includes('.xcworkspace'));
+    if (workspaceCollection) {
+        projectName = fs.readdirSync(project_dir).find(d => d.includes('.xcworkspace')).replace('.xcworkspace', '');
+    }
     const xcBuildConfiguration = xcodeproj.pbxXCBuildConfigurationSection();
-    const plist_file_entry = _.find(xcBuildConfiguration, entry => entry.buildSettings && entry.buildSettings.INFOPLIST_FILE);
+    const plist_file_entry = _.find(xcBuildConfiguration, function (entry) {
+        return (
+            entry.buildSettings &&
+            entry.buildSettings.INFOPLIST_FILE &&
+            !(projectName && entry.buildSettings.INFOPLIST_FILE.includes(`${projectName}/${projectName}-Info.plist`))
```

Comment:
Needs to be corrected (I got it wrong in my suggestion):

```suggestion
            !(projectName && !entry.buildSettings.INFOPLIST_FILE.includes(`${projectName}/${projectName}-Info.plist`))
```

or better yet:

```suggestion
            (!projectName ||
                entry.buildSettings.INFOPLIST_FILE.includes(`${projectName}/${projectName}-Info.plist`))
```